### ISSUE TITLE
macos Display webview in the current page without opening a new window

### ISSUE THREAD

**Issue by spring321** [pichillilorenzo/flutter_inappwebview#2096]
macos  How do I present the webview without opening a new window or using the system browser

**Comment by tfabacher-dp** [comment]
Yes, we are experiencing the problem of just showing a webpage on the Mac as a Desktop App. I have seen the videos of it loading on your site and we followed the instructions for the XCode settings, but we can NOT get a webpage to load on a Flutter MacOS desktop app.

Please Please...any suggestions or samples would be most appreciated!!

--Todd
